web/src/feedback_widget.ts

```diff
@@ -157,17 +157,16 @@ export function show(opts: FeedbackWidgetOptions): void {
 
     meta.$container = $("#feedback_container");
 
```

Comment:
Can you explain the need for this commit?

Reply:
I was working locally on this PR before the commit d5eb99724294c3cdfae1a2860f75f03f8ae54fd1 was merged. I made these changes prior to rebasing.

Now these changes are not required.